plugins/janus_sip.c

```diff
@@ -3040,7 +3103,8 @@ static void *janus_sip_handler(void *data) {
 			json_object_set_new(result, "event", json_string("hangingup"));
 		} else if(!strcasecmp(request_text, "recording")) {
 			/* Start or stop recording */
-			if(!(session->status == janus_sip_call_status_inviting || session->status == janus_sip_call_status_incall)) {
+			if(!(session->status == janus_sip_call_status_inviting || /* Presume it makes sense to start recording with early media? */
+				janus_sip_call_is_established(session))) {
```

Comment:
Since this part of the `if` is on a second line, it needs to be indented with one more tab per our code style.

Reply:
Scanned and fixed all occurrences. BTW, there is seems to be inconsistency in spacing between if and () throughout the code. Anyway, changed all ifs so that it's not followed by a space.